web/views/days_view.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Janitor.DaysView do
+  use Janitor.Web, :view
+
+  def render("days.json", %{days: days}) do
+    days |> Enum.map(fn (day) -> 
```

Comment:
Please consider factoring it out to a private function:

```elixir
def render(days) do
  days |> Enum.map(&pick_fields\1)
end

defp pick_fields(day)
  day
    |> Map.from_struct
    |> Map.take([:id, :date, :working, :user_id])
end
```

Given that this function is used in the spec maybe it can even be factored out to some common module and made public.